content/applications/finance/fiscal_localizations/united_states.rst

```diff
@@ -0,0 +1,746 @@
+
+.. image:: united_states/us-l10n-nacha-settings.png
+   :align: center
+   :alt: NACHA (National Automated Clearing House Association) configuration settings on Odoo.
+
+.. note::
+   The following |NACHA| configuration information is normally provided by the company's financial
+   institution once they have been approved to send payments via their account.
+
+Under the section labeled, :guilabel:`NACHA configuration` are the fields required to generate a
+|NACHA| compatible |ACH| file to send to a company's bank. First, enter the *routing number* of the
+financial institution in the field labeled, :guilabel:`Immediate Destination`. This information is
+widely available on the Internet and generally varies by bank location. This number is usually
+provided during the initial account setup.
+
+Next, enter the registered name of the financial institution in the field called,
+:guilabel:`Destination`. This information will be provided by the bank or credit union.
+
+Following the :guilabel:`Destination` field is the :guilabel:`Immediate Origin` field. Enter the
+9-digit company ID (Employer Identification Number (EIN)) into this field. This information is
+provided by the financial institution.
+
+Next, the :guilabel:`Company Identification` number should be entered. This is a 10-digit number
+made up of the 9-digit company ID (Employer Identification Number (EIN)), with an additional number
+at the start of the sequence. This number is often a `1`. Check with the financial institution
+should this first number differ to verify that it is correct. This number is provided for |ACH|
+approved accounts.
+
+The :guilabel:`Originating DFI Identification` number is next, this Depository Financial Institution
+(DFI) field contains an assigned 8-digit number. Consult the financial institution for this number.
+
+.. important::
+   Enter the above numbers exactly as the company's financial institution (bank or credit union) has
+   provided them. Failing to do so, would be detrimental to the successful  |NACHA| configuration.
+
+.. image:: united_states/us-l10n-nacha-dropdown.png
+   :align: center
+   :alt: NACHA settings with the standard entry class code drop-down menu highlighted.
+
+There are two options for the next field: :guilabel:`Standard Entry Class Code`. Select the
+drop-down menu to the right of the field and pick either :guilabel:`Corporate Credit or Debit (CCD)`
+or :guilabel:`Prearranged Payment and Deposit (PPD)`. Again, this information will be provided by
+the financial institution. By default :guilabel:`Corporate Credit or Debit (CCD)` is selected.
+
+Finally, the last option is for :guilabel:`Generated Balanced Files`. Tick the checkbox to the right
+of the field to enable :guilabel:`Generated Balanced Files`. Consult the company's accountant or
+financial advisor to make an informed decision for this field.
+
+Manually :icon:`fa-cloud-upload` :guilabel:`(cloud upload)` save the configuration or navigate away
```

Comment:
Wording, to more clearly tie the result to the clickable action
The prior version places the icon label next to the main action verb, which risks confusing readers.

```suggestion
Manually save the configuration by clicking the :icon:`fa-cloud-upload` :guilabel:`(cloud upload)` icon, or navigate away
```